jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -419,6 +419,8 @@ void java_bytecode_convert_methodt::convert(
     get_method_identifier(class_symbol.name, m);
 
   method_id=method_identifier;
+  set_owning_class(
```

Comment:
:pick: Lower down (line 537) you have a writeable ref for this symbol already which is used to update the type. It might be nicer to keep the mutation in one place and do this call down there

Reply:
I tried putting it lower down and things didn't work properly. I guess it must be read back somewhere between the two locations.